### DISCUSSION TITLE
Difference between `:NvimTreeFocus` and `:NvimTreeOpen`?

### DISCUSSION THREAD

**Discussion by SoheilSalmani** [nvim-tree/nvim-tree.lua#2554] [Q&A] [answered]
I wonder what's the difference between `:NvimTreeFocus` and `:NvimTreeOpen`. It seems to me that they have exactly the same behavior. Is there any reason to use one instead of the other?

**Comment by alex-courtis** [comment, marked as answer]
Out of the box they are basically the same.

Open, however, takes a path argument and allows for customisation of behaviour. See `:help nvim-tree-api.tree.open()`

**Reply by gegoune** [reply]
Do we need both of them then? :)

**Reply by alex-courtis** [reply]
Sounds like we don't. Both commands should call `open()`

However... if we remove `focus()` it might break API users.

**Reply by gegoune** [reply]
Could just leave shim and soft deprecate it for now.

**Reply by alex-courtis** [reply]
That cleaned up nicely: #2558 

Review / test most gratefully appreciated!

**Reply by gegoune** [reply]
Quickly tested and approved. Thank you!

**Comment by alex-courtis** [comment]
Updated help with more info: https://github.com/nvim-tree/nvim-tree.lua/pull/2556
